src/algebra/star/basic.lean

```diff
@@ -394,6 +395,17 @@ begin
   rw [ring.inverse_non_unit _ ha, ring.inverse_non_unit _ (mt is_unit_star.mp ha), star_zero],
 end
 
+instance invertible.star {R : Type*} [monoid R] [star_monoid R] (r : R) [invertible r] :
+  invertible (star r) :=
+{ inv_of := star (⅟r),
+  inv_of_mul_self := by rw [←star_mul, mul_inv_of_self, star_one],
+  mul_inv_of_self := by rw [←star_mul, inv_of_mul_self, star_one] }
+
+lemma star_inv_of {R : Type*} [monoid R] [star_monoid R] (r : R)
+  [invertible r] [invertible (star r)] :
```

Comment:
is this a compatibility lemma (say for other instances of `invertible (star r)`)? I think it's worth noting this if so

Reply:
Can you elaborate on what you're suggesting here?

Reply:
I thought about this more and I think it's fine, actually